Octokit.Reactive/Clients/ObservableIssueCommentsClient.cs

```diff
@@ -87,7 +87,7 @@ public IObservable<IssueComment> GetAllForRepository(string owner, string name,
             Ensure.ArgumentNotNullOrEmptyString(name, "name");
             Ensure.ArgumentNotNull(options, "options");
 
-            return _connection.GetAndFlattenAllPages<IssueComment>(ApiUrls.IssueComments(owner, name), null, AcceptHeaders.ReactionsPreview, options);
+            return GetAllForRepository(owner, name, new IssueCommentRequest(), ApiOptions.None);
```

Comment:
This should be passing through the `options` variable rather than `ApiOptions.None`